.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -0,0 +1,89 @@
+name: Release checklist
+description: Checklist for each release. This template is only for the core team.
+title: "[Release] Element Android v "
```

Comment:
There is an extra space after the "v" which will annoy me each time I create such issue :)